recipes/liburing/all/patches/liburing-2.4-disable_samples_and_tests.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/Makefile b/Makefile
+index 73d021c..74aa8d7 100644
+--- a/Makefile
++++ b/Makefile
+@@ -8,8 +8,6 @@ default: all
+ 
+ all:
+ 	@$(MAKE) -C src
+-	@$(MAKE) -C test
+-	@$(MAKE) -C examples
```

Comment:
What's the difference between the 2.4 patch and the 2.2 one? Not seeing a difference.

Also, this might be an okay use for `replace_in_file` instead of patching, since the changes are fairly straightforward and only differ in context.

Reply:
At first, I wanted to have a single patch to remove tests and examples build. The patch applies correctly on all versions on my side but for some reason, it fails in the CI.
I then decided to produce a patch for each version and remove duplicates (md5sum).
I don't know about `replace_in_file`. I will have a look.

Reply:
Would be essentially something like `replace_in_file(self, os.path.join(self.source_folder, "Makefile"), "@$(MAKE) -C test", "")`